pwnagotchi/plugins/default/grid.py

```diff
@@ -36,6 +37,10 @@ def parse_pcap(filename):
         # /root/handshakes/BSSID.pcap
         essid, bssid = '', net_id
 
+    mac_re = re.compile('[0-9a-fA-F]{12}')
```

Comment:
shouldn't this also include : ? i don't think this will match valid BSSIDs. it should be `[0-9a-fA-F:]{17}`

Reply:
I'm matching the on-disk filenames with this regex, which (bettercap?) seems to poop out in the ssid_aabbccddeeff.pcap format.

Reply:
oh right, my bad!